inductiva/_cli/cmd_tasks/kill.py

```diff
@@ -34,13 +34,17 @@ def kill_task(args):
                 "task_id": task.id
             } for task in get_all(status=status)])
 
+    if not tasks:
+        print("There are not tasks to kill.")
+        return 1
```

Comment:
Could we please document what is the meaning of each return code ? @Pbarbosa92 @rcvalerio

Reply:
> Could we please document what is the meaning of each return code ? @Pbarbosa92 @rcvalerio

There is no specific meaning. If it all goes well we return 0. Any kind of error we return non 0 (1 in this case) but there is no error codes at this time.

Reply:
I believe the return codes are from `argParse` and they are common in all our client commands.

`0`: Indicates that the program executed successfully without errors. This is the standard convention for success in Unix-like operating systems.
`1`: Indicates that the program encountered an error. This is a common convention for general errors, but specific programs might use other non-zero values to signify different types of errors.

Reply:
Ok, I see, thanks -- this integer return codes are being used exclusively on the CLI (command line interface), which makes sense because it's returning to the shell:
https://github.com/search?q=repo%3Ainductiva%2Finductiva%20%22return%201%22&type=code

In the Python package logic, functions and methods are not using that pattern (correctly), because in Python you would typically return some richer type, or potentially raise an exception if an error occurs.

Cool, thanks for the clarification.